packages/plugin-session-replay-browser/src/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const DEFAULT_EVENT_PROPERTY_PREFIX = '[Amplitude]';
+
+export const DEFAULT_SESSION_REPLAY_PROPERTY = `${DEFAULT_EVENT_PROPERTY_PREFIX} Session Recorded`;
+export const DEFAULT_SESSION_START_EVENT = 'session_start';
```

Comment:
Should we reuse [this](https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/analytics-browser/src/constants.ts#L9-L10) and DEFAULT_EVENT_PREFIX?

Reply:
for SDKs, i'd lean on exposing less than more so there's less surface area for breaking changes. i think duplicating was a good call